main.js

```diff
@@ -0,0 +1,98 @@
+const form = document.querySelector('.add-book');
+const bookTitle = document.querySelector('.title');
+const bookAuthor = document.querySelector('.author');
+const bookList = document.querySelector('.book-list');
+class Storage {
+  constructor() {
+    this.collection = [];
+  }
+
+  static addCollection(newBook) {
+    this.collection.push(newBook);
```

Comment:
- [ ] OPTIONAL: Array `concat` method is preferred over `push` method for the sake of data integrity as it does not alter the original array. :+1: